tests/Feature/CommandsTest.php

```
@@ -0,0 +1,82 @@
+            'vendor-cleanup:lang' => 'Report which published lang files differ from their vendor originals',
+            'vendor-cleanup:view' => 'Report which published view files differ from their vendor originals',
+        ];
+
+        foreach ($commands as $commandName => $expectedDescription) {
+            $output = $this->artisan('list', ['--format' => 'json'])
+                ->run();
+
+            $this->artisan('list')
+                ->expectsOutputToContain($commandName);
+        }
+    }
```

PHPMD (2.15.0)

The method test_commands_have_correct_descriptions is not named in camelCase. (undefined)

(CamelCaseMethodName)